config.go

```diff
@@ -1616,6 +1622,35 @@ func ValidateConfig(cfg Config, interceptor signal.Interceptor, fileParser,
 		cfg.DB.Bolt.NoFreelistSync = !cfg.SyncFreelist
 	}
 
+	// Parse any extra sqlite pragma options that may have been provided
+	// to determine if they override any of the defaults that we will
+	// otherwise add.
+	sqlitePragmaOpts := make(
+		map[string]bool, len(cfg.DB.Sqlite.PragmaOptions),
+	)
+	for _, option := range cfg.DB.Sqlite.PragmaOptions {
+		pair := strings.Split(option, "=")
+		if len(pair) != 2 { //nolint:gmnd
```

Comment:
I think this needs a bare `//nolint` comment, this one doesn't seem to be working.

Reply:
cool, trying that now

Reply:
also looks like the linter is catching 2 completely unrelated errors....

Reply:
it is strangely still failing 🙃

Reply:
LOL im spelling it wrong 😂